app/models/setting/remote_execution.rb

```diff
@@ -22,7 +22,10 @@ def self.load_defaults
                  'root'),
         self.set('remote_execution_effective_user_method',
                  N_('What command should be used to switch to the effective user. One of %s') % SSHExecutionProvider::EFFECTIVE_USER_METHODS.inspect,
-                 'sudo'),
+                 'sudo',
+                 'remote_execution_effective_user_method',
+                 nil,
+                 { :collection => Hash[SSHExecutionProvider::EFFECTIVE_USER_METHODS.map{|method| [method, method]}]}),
```

Comment:
Thanks, looks good.  The proc is somewhat useless here I agree, I wish it was optional but oh well.